src/js/components/SearchBar/MenuAction.tsx

```diff
@@ -2,18 +2,18 @@ import {useBrimApi} from "app/core/context"
 import React from "react"
 import {useDispatch, useSelector} from "react-redux"
 import Tabs from "src/js/state/Tabs"
+import Viewer from "src/js/state/Viewer"
 import {reactElementProps} from "../../../../test/playwright/helpers/integration"
 import ThreeDotsIcon from "../../icons/ThreeDotsIcon"
 import open from "../../lib/open"
 import Modal from "../../state/Modal"
-import Tab from "../../state/Tab"
 import PopMenuPointy from "../PopMenu/PopMenuPointy"
 import InputAction from "./InputAction"
 
 export default function MenuAction() {
   const dispatch = useDispatch()
   const api = useBrimApi()
-  const isFetching = useSelector(Tab.isFetching)
+  const isFetching = useSelector(Viewer.isFetching)
```

Comment:
I decided to only care about the viewer search instead of the viewer and the chart.